src/llama-arch.cpp

```diff
@@ -462,6 +463,24 @@ static const std::map<llm_arch, std::map<llm_tensor, const char *>> LLM_TENSOR_N
             { LLM_TENSOR_CLS_OUT,         "cls.output" },
         },
     },
+    {
+        LLM_ARCH_DISTIL_BERT,
+        {
+            { LLM_TENSOR_TOKEN_EMBD,      "token_embd" },
+            { LLM_TENSOR_TOKEN_EMBD_NORM, "token_embd_norm" },
+            { LLM_TENSOR_POS_EMBD,        "position_embd" },
```

Comment:
```suggestion
            { LLM_TENSOR_TOKEN_EMBD_NORM, "token_embd_norm" },
            { LLM_TENSOR_TOKEN_TYPES,     "token_types" }, // not used
            { LLM_TENSOR_POS_EMBD,        "position_embd" },
```
Oh, sorry, since we changed it to `TENSOR_NOT_REQUIRED` it has to be here.

Reply:
On second though, scratch that, we can change DistilBert to Bert arch and remove all this.

Reply:
You were too fast. :)

Reply:
Not sure whether it's necessary or not, but I added. If this llama-cpp's convention?